app/controllers/mechanics_controller.rb

```diff
@@ -0,0 +1,10 @@
+class MechanicsController < ApplicationController
+  def show
+    @mechanic = Mechanic.find(params[:mechanic_id])
+  end
+
+  def create
+    MechanicRide.create!(mechanic_id:params[:mechanic_id], ride_id:params[:ride_id])
+    redirect_to "/mechanics/#{params[:mechanic_id]}"
+  end
```

Comment:
This logic looks good. I think that this method should go into a `MechanicRidesController` in a `create` action since that's the kind of object we are trying to create. I think this because, what if you ever wanted to add functionality to create Mechanics? The create action would then be taken up!